purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -153,6 +173,9 @@ open class PurchasesConfiguration(builder: Builder) {
             this.dangerousSettings = dangerousSettings
         }
 
+        /**
+         * Creates a [PurchasesConfiguration] instance with the specified properties.
+         */
         open fun build(): PurchasesConfiguration {
```

Comment:
This could be final?

Reply:
It might be open for tests? let me check

Reply:
actually, it looks like it can be final, but.... that would be a breaking change 😕

Reply:
Oh right. It's fine.